testsuite/tests/contrib/lkt_semantic/array_type_concrete/test.lkt

```diff
@@ -0,0 +1,15 @@
+val i: Int = 1
+# TODO: should infer type from i
+val a = [i, 2, 3]
```

Comment:
All array values in this file are called `a`, which should probably be invalid in the future. Could you rename them? For intsance a1, a2, …

Reply:
Sure!